src/main/java/model/Player.java

```diff
@@ -0,0 +1,90 @@
+package model;
+
+
+import exception.InvalidMoneyAmountException;
+
+public class Player extends Person{
+    private final Wallet wallet;
+    private Bet bet;
+    private int playerNumber;
+
+    public Player(){
+        wallet = new Wallet(100);
+    }
+
+    /**
+     * @return the balance of the player's wallet
+     */
+    public int balance(){
+        return wallet.getAmount();
+    }
+
+    /**
+     * withdraws a certain amount of money from the player's wallet
+     * @param amount: int
+     * @pre amount >= 0
+     * @throws InvalidMoneyAmountException: if not enough money or invalid amount
+     */
+    public void withdraw(int amount) throws InvalidMoneyAmountException {
+        if (amount < 0) {
```

Comment:
This is confusing, for `withdraw` you put a check here. For `put`, you added the check in Wallet. I'd do everything in Wallet.